portfolio/src/main/webapp/script.js

```diff
@@ -24,20 +24,33 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+function addMyCode() {
+    addMyComments();
+    login();
+    console.log("success");
```

Comment:
Nit: technically you want to avoid leaving console logs in your master code to avoid surfacing debug info to viewers (unless you want to notify them about some error, etc)